clients/js/src/CloudClient.ts

```diff
@@ -0,0 +1,48 @@
+
+// create a cloudclient class that takes in an api key and an optional database
+// this should wrap ChromaClient and specify the auth scheme correctly
+
+import { ChromaClient } from "./ChromaClient";
+
+interface CloudClientParams {
+    apiKey?: string;
+    database?: string;
+    cloudHost?: string;
+    cloudPort?: string;
+    // enableSsl?: boolean;
+}
+
+class CloudClient extends ChromaClient{
+    // private enableSsl: boolean;
```

Comment:
Do we need this? Seems like we expect protocol to be passed as part of the host (or default to `http` for `localhost`, presumably `https` for cloud)

Reply:
yea - i wasnt sure about this? the python client had it, so i had stubbed it out. im fine removing it. just did!